apps-rendering/src/components/Callout/calloutBlock.tsx

```diff
@@ -32,6 +32,7 @@ const CalloutBlock: FC<CalloutBlockProps> = ({
 	format,
 	description,
 }): ReactElement => {
+	const id = `${heading.replace(/[\s_]+/g, '-')}-callout`;
```

Comment:
Just a question, do you think this can be passed as a parameter? because the same regex is happening in the index.tsx